script/e2e-test.sh

```diff
@@ -134,25 +137,44 @@ pushChart() {
       --set assetsvc.replicaCount=1 \
       --set dashboard.replicaCount=1 \
-      --set postgresql.replication.enabled=false
+      --set postgresql.replication.enabled=false \
+      --set ingress.enabled=true  \
+      --set ingress.hostname=localhost  \
+      --set ingress.tls=true  \
+      --set authProxy.enabled=true \
+      --set authProxy.provider=oidc \
+      --set authProxy.clientID=default \
+      --set authProxy.clientSecret=ZXhhbXBsZS1hcHAtc2VjcmV0 \
+      --set authProxy.cookieSecret=bm90LWdvb2Qtc2VjcmV0Cg== \
+      --set authProxy.additionalFlags[0]="--oidc-issuer-url=https://${DEX_IP}:32000" \
+      --set authProxy.additionalFlags[1]="--scope=openid email groups audience:server:client_id:second-cluster audience:server:client_id:third-cluster" \
+      --set authProxy.additionalFlags[2]="--ssl-insecure-skip-verify=true" \
+      --set authProxy.additionalFlags[3]="--redirect-url=http://kubeapps-ci.kubeapps/oauth2/callback" \
+      --set authProxy.additionalFlags[4]="--cookie-secure=false" \
+      --set authProxy.additionalFlags[5]="--cookie-domain=kubeapps-ci.kubeapps" \
+      --set authProxy.additionalFlags[6]="--whitelist-domain=kubeapps-ci.kubeapps" \
+      --set authProxy.additionalFlags[7]="--set-authorization-header=true" \
+      --set clusters[0].name=default \
+      --set clusters[1].name=second-cluster \
+      --set clusters[1].apiServiceURL="https://${ADDITIONAL_CLUSTER_IP}:6443" \
+      --set clusters[1].insecure=true \
+      --set clusters[1].serviceToken=ZXlKaGJHY2lPaUpTVXpJMU5pSXNJbXRwWkNJNklsbHpiSEp5TlZwM1QwaG9WSE5PYkhVdE5GQkRablY2TW0wd05rUmtMVmxFWVV4MlZEazNaeTEyUmxFaWZRLmV5SnBjM01pT2lKcmRXSmxjbTVsZEdWekwzTmxjblpwWTJWaFkyTnZkVzUwSWl3aWEzVmlaWEp1WlhSbGN5NXBieTl6WlhKMmFXTmxZV05qYjNWdWRDOXVZVzFsYzNCaFkyVWlPaUprWldaaGRXeDBJaXdpYTNWaVpYSnVaWFJsY3k1cGJ5OXpaWEoyYVdObFlXTmpiM1Z1ZEM5elpXTnlaWFF1Ym1GdFpTSTZJbXQxWW1WaGNIQnpMVzVoYldWemNHRmpaUzFrYVhOamIzWmxjbmt0ZEc5clpXNHRjV295Ym1naUxDSnJkV0psY201bGRHVnpMbWx2TDNObGNuWnBZMlZoWTJOdmRXNTBMM05sY25acFkyVXRZV05qYjNWdWRDNXVZVzFsSWpvaWEzVmlaV0Z3Y0hNdGJtRnRaWE53WVdObExXUnBjMk52ZG1WeWVTSXNJbXQxWW1WeWJtVjBaWE11YVc4dmMyVnlkbWxqWldGalkyOTFiblF2YzJWeWRtbGpaUzFoWTJOdmRXNTBMblZwWkNJNkltVXhaakE1WmpSakxUTTRNemt0TkRJME15MWhZbUptTFRKaU5HWm1OREZrWW1RMllTSXNJbk4xWWlJNkluTjVjM1JsYlRwelpYSjJhV05sWVdOamIzVnVkRHBrWldaaGRXeDBPbXQxWW1WaGNIQnpMVzVoYldWemNHRmpaUzFrYVhOamIzWmxjbmtpZlEuTnh6V2dsUGlrVWpROVQ1NkpWM2xJN1VWTUVSR3J2bklPSHJENkh4dUVwR0luLWFUUzV5Q0pDa3Z0cTF6S3Z3b05sc2MyX0YxaTdFOUxWRGFwbC1UQlhleUN5Rl92S1B1TDF4dTdqZFBMZ1dKT1pQX3JMcXppaDV4ZlkxalFoOHNhdTRZclFJLUtqb3U1UkRRZ0tOQS1BaS1lRlFOZVh2bmlUNlBKYWVkc184V0t3dHRMMC1wdHpYRnBnOFl5dkx6N0U1UWdTR2tjNWpDVXlsS0RvZVRUaVRSOEc2RHFHYkFQQUYwREt0b3MybU9Geno4SlJYNHhoQmdvaUcxVTVmR1g4Z3hnTU1SV0VHRE9kaGMyeXRvcFdRUkRpYmhvaldNS3VDZlNua09zMDRGYTBkYmEwQ0NTbld2a29LZ3Z4QVR5aVVrWm9wV3VpZ1JJNFd5dDkzbXhR
 }
 
-# Operators are not supported in GKE 1.14 and flaky in 1.15
-if [[ -z "${GKE_BRANCH-}" ]]; then
-  installOLM 0.16.1
-fi
```

Comment:
I totally omitted the fact that this file is also used in GKE...  I remove it because I already installed it in the local_e2e test so that the "setting up the environment" and the "actual e2e test" scenarios are differenciated. 
Next week I'll revisit it. Thanks for pointing this out!